ci-operator/templates/cluster-launch-installer-e2e.yaml

```diff
@@ -195,26 +194,19 @@ objects:
         mkdir /tmp/cluster
         cp /etc/openshift-installer/* /tmp/cluster/
 
-        export NAME=${INSTANCE_PREFIX}
-        (
-          export EMAIL=test@ci.openshift.io
-          export PASSWORD=$( date +%s | sha256sum | base64 | head -c 32 ; echo )
-          export SSH_KEY="$( cat /etc/openshift-installer/ssh-publickey )"
-          export EXPIRATION_DATE=$( date -d '4 hours' --iso=minutes --utc )
-          envsubst </etc/openshift-installer/openshift.yaml >/tmp/cluster/inputs.yaml
-        )
         mkdir /tmp/artifacts/installer
-        cp /tmp/cluster/inputs.yaml /tmp/artifacts/installer/
-
-        echo "Invoking installer ..."
-        
         cd /tmp/cluster
-        tectonic init --config=inputs.yaml
-        mv -f ${NAME}/* /tmp/cluster/
-        cp config.yaml internal.yaml /tmp/artifacts/installer/
 
-        tectonic install --dir=. --log-level=debug
-        cp terraform.tfvars /tmp/artifacts/installer/
+        echo "Invoking installer ..."
+        OPENSHIFT_INSTALL_CLUSTER_NAME="${INSTANCE_PREFIX}" \
+        OPENSHIFT_INSTALL_BASE_DOMAIN=origin-ci-int-aws.dev.rhcloud.com \
```

Comment:
Be aware that at some point in the next month or so we'll likely need to move this back out of the job definition and make parametrizable for different teams and content streams.  But it's fine for now.